lexicant.html

```diff
@@ -12,7 +15,7 @@
     <div id="message"></div>
 </body>
 
-<script src="LetterBitMapTrie.js"></script>
+<script src="/LetterBitMapTrie.js"></script>
```

Comment:
I made the URLs absolute for consistency, let me know if you have a preference for relative ones and I will instead change "/style.css" to "style.css" above.

Reply:
I have no strong preference, just must have overlooked this